tsconfig.json

```diff
@@ -14,6 +14,15 @@
     "isolatedModules": true,
     "jsx": "preserve"
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "firebaseClient.ts",
+    "firebaseAdmin.ts",
+    "components/Container.tsx",
+    "pages/index.tsx",
+    "pages/_app.tsx"
```

Comment:
no need to explicitly include each files, the wild-card format `**/*.ts` and `**/*.tsx` covers all these files already

Reply:
Well, i haven't included these files....every typescript file is automatically getting added there.

Reply:
🤔